lib/docs/facts.md

```diff
@@ -0,0 +1,1977 @@
+## `architecture`
```

Comment:
Yeah I wasn't exactly sure where it came from, but I also wasn't sure if it was supposed to be ignored or not. I'll add it.